Modules/rat_facts.py

```diff
@@ -0,0 +1,86 @@
+"""
+rat_facts.py - Managing Facts since 2018
+
+Copyright (c) 2018 The Fuel Rat Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+
+This module is built on top of the Pydle system.
+
+"""
+from utils.ratlib import Singleton
+from Modules.database_manager import DatabaseManager
+
+
+class FactsManager(metaclass=Singleton):
+
+    def __init__(self):
+        """
+        retrieves the DBM object. should this fail, it will set the variable to None,
+            failing all subsequent calls
+        """
+        try:
+            self.dbm = DatabaseManager()
+        except:
+            self.dbm = None
```

Comment:
This is not at all a safe way to do this for two reasons:
* Bare except clauses are a bad idea in most circumstances and your IDE should tell you this. You should only catch those exception which you expect to be thrown and whish to handle.
* If there is something wrong and the DBM can't be initialized, this constructor will run through quietly as if all were fine, leaving the fire to be noticed only when someone attempts to use a fact. Errors should be noticed as soon as possible, please raise an exception here if at all possible.

Reply:
This may be a leftover to prevent pytest from failing on collection.